PS8lite.Rmd

```diff
@@ -201,9 +211,21 @@ Fit one Random Forest model `model_rf` using any approach you want.
 Using `model_rf` make predictions and submit to Kaggle.
 
 ```{r}
+#predict on test data
+test <- test %>% 
+  mutate(
+    logSalePrice_hat_rf = predict(object = model_rf, newdata = test),
+    SalePrice_hat_rf = exp(logSalePrice_hat_rf) -1
+  )
+
+#write submission
+ submission_rf <- test %>%
+   mutate(SalePrice = SalePrice_hat_rf) %>%
+   select(Id, SalePrice)
+ write_csv(submission_rf, "data/submission_rf.csv")
 
 ```
 
 
-![](images/score_screenshot_naive.png){ width=100% }
+![](images/score_screenshot.png){ width=100% }
```

Comment:
Had this been a real GitHub PR, you need to submit 1score_screenshot.png1 as part of your assignment or I can't reproduce your work! Don't worry about it here.